modules/casper.js

```diff
@@ -2513,6 +2513,7 @@ Casper.prototype.newPage = function newPage() {
     if (utils.isObject(this.options.viewportSize)) {
         this.page.viewportSize = this.options.viewportSize;
     }
+    this.page.clearMemoryCache()
```

Comment:
uncaughtError: TypeError: 'undefined' is not a function (evaluating 'this.page.clearMemoryCache()')